File: patterns/c-patterns/pattern119.c
// Pattern 119: ...

// *
// *
// **
// **
// ***
// ***


#include <stdio.h>

int main()
{
    int n = 3;
    int i, j, c;
    
    for(i = 1; i<= n; i++)
    {
        for( c= n- 1; c>0; c--)
        {
            for(j = 0; j<i; j++)
            {
                printf("*");
            }
            printf("\n");
        }
    }
    return 0;
}

File: patterns/javascript-patterns/pattern119.js
// Pattern 119: ...

// *
// *
// **
// **
// ***
// ***

let n = 3;

for (let i = 1; i <= n; i++) {
    for (let c = n - 1; c > 0; c--) {
        let row = '';
        for (let j = 0; j < i; j++) {
            row += '*';
        }
        console.log(row);
    }
}
